features/promotion/applying_catalog_promotions/reapplying_multiple_catalog_promotions_after_editing_their_channels.feature

```diff
@@ -0,0 +1,38 @@
+@applying_catalog_promotions
+Feature: Reapplying multiple catalog promotions after editing their channels
+    In order to have proper discounts in different channels
+    As a Store Owner
+    I want to have discounts reapplied in product catalog once the channels of catalog promotion changes
+
+    Background:
+        Given the store operates on a channel named "Web-US" with hostname "web-us"
+        And the store operates on another channel named "Web-GB" with hostname "web-gb"
+        And the store has a "T-Shirt" configurable product
+        And this product has "PHP T-Shirt" variant priced at "$100.00" in "Web-US" channel
+        And "PHP T-Shirt" variant priced at "$30.00" in "Web-GB" channel
+        And this product is available in "Web-US" channel and "Web-GB" channel
+        And there is a catalog promotion "Winter sale" available in "Web-US" channel that reduces price by "30%" and applies on "PHP T-shirt" variant
+        And there is a another catalog promotion "Christmas sale" available in "Web-US" channel and "Web-GB" channel that reduces price by "50%" and applies on "PHP T-shirt" variant
+        And I am logged in as an administrator
+
+    @api @todo
+    Scenario: Removing only modified catalog promotion after removing its channel
+        When I make "Winter sale" catalog promotion unavailable in the "Web-US" channel
+        Then the visitor should see "$50.00" as the price of the "T-Shirt" product in the "Web-US" channel
+        And the visitor should see "$100.00" as the original price of the "T-Shirt" product in the "Web-US" channel
+
+    @api @todo
+    Scenario: Reapplying catalog promotion after adding new channel to them
+        When I make "Winter sale" catalog promotion available in the "Web-GB" channel
+        Then the visitor should see "$35.00" as the price of the "T-Shirt" product in the "Web-GB" channel
+        And the visitor should see "$100.00" as the original price of the "T-Shirt" product in the "Web-GB" channel
+        And the visitor should still see "$35.00" as the price of the "T-Shirt" product in the "Web-US" channel
+        And the visitor should still see "$100.00" as the original price of the "T-Shirt" product in the "Web-US" channel
+
+    @api @todo
+    Scenario: Reapplying catalog promotion after switching availability in channels
```

Comment:
This scenario looks the same as the one from `features/promotion/applying_catalog_promotions/reapplying_catalog_promotions_after_editing_its_channels.feature:32`
I would be for modifying this scenario to have a store with 3 channels, a variant with 2 catalog promotions applied and one them will be modified to change one of its channel

Reply:
I'm resolving as it was an invalid comment 🎉